lang/ilias_sk.lang

```diff
@@ -7943,7 +7943,7 @@ chatroom_adm#:#client_settings#:#Client settings###06 08 2011 new variable
 chatroom_adm#:#https#:#HTTPS###06 08 2011 new variable
 chatroom_adm#:#chat_enabled#:#Enable chat###06 08 2011 new variable
 chatroom_adm#:#enable_osd#:#Enable On-Screen notifications###06 08 2011 new variable
-chatroom_adm#:#osd_intervall#:#Refreshinterval (seconds)###06 08 2011 new variable
+chatroom_adm#:#osd_interval#:#Refreshinterval (seconds)###06 08 2011 new variable
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.